intro-HPC/ch_web_portal.tex

```diff
@@ -0,0 +1,217 @@
+For more information, see the sections below as well as
+\url{https://www.osc.edu/resources/online_portals/ondemand/job_management}.
+
+\subsubsection{Active jobs}
+
+To get an overview of all your currently active jobs,
+use the \emph{Active Jobs} menu item under \emph{Jobs}.
+
+A new browser tab will be opened that shows all your current queued and/or running jobs:
+
+\begin{center}
+    \includegraphics*[scale=0.5]{ood_active_jobs}
+\end{center}
+
+You can control which jobs are shown using the \emph{Filter} input area,
+or select a particular cluster from the drop-down menu \emph{All Clusters}, both at the top right.
+
+Jobs that are still queued or running can be deleted using the red button on the right.
+
+Completed jobs will also be visible in this interface, but only for a short amount of time after they have stopped running.
+
+For each listed job, you can click on the arrow ($>$) symbol to get a detailed overview of that job,
+and get quick access to the corresponding output directory (via the \emph{Open in File Manager} and \emph{Open in Terminal} buttons at the bottom of the detailed overview).
+
+\subsubsection{Job composer}
+
+To submit new jobs, you can use the \emph{Job Composer} menu item under \emph{Jobs}.
+This will open a new browser tab providing an interface to create new jobs:
+
+\begin{center}
+    \includegraphics*[scale=0.35]{ood_job_composer}
+\end{center}
+
+This extensive interface allows you to create jobs from one of the available templates,
+or by copying an existing job.
+
+You can carefuly prepare your job and the corresponding job script via the \emph{Job Options} button
+and by editing the job script (see lower right).
+
+Don't forget to actually submit your job to the system via the green \emph{Submit} button!
+
+\paragraph{Job templates}
+
+In addition, you can inspect provided job templates, copy them or even create your own templates via the \emph{Templates} button on the top:
+
+\begin{center}
+    \includegraphics*[scale=0.35]{ood_job_templates}
+\end{center}
+
+\subsection{Shell access}
+
+Through the \emph{Shell Access} button that is available under the \emph{Clusters} menu item,
+you can easily open a terminal (shell) session into your VSC account, straight from your browser!
+
+\begin{center}
+    \includegraphics*[scale=0.5]{ood_shell}
+\end{center}
+
+Using this interface requires being familiar with a Linux shell environment (see Appendix~\ref{ch:useful-linux-commands}).
+
+To exit the shell session, type \lstinline|exit| followed by \emph{Enter} and then close the browser tab.
+
+\subsection{Interactive applications}
+
+\subsubsection{Graphical desktop environment}
+
+To create a graphical desktop environment, use on of the \emph{desktop on ... node} buttons under \emph{Interactive Apps} menu item. For example:
+
+\begin{center}
+    \includegraphics*[scale=0.35]{ood_launch_desktop}
+\end{center}
+
+You can either start a desktop environment on a login node (which should start almost instantly)
```

Comment:
I would leave the "(which should start almost instantly)" out

Reply:
second that!

Reply:
dropped the `should start` part + clarified a bit